qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/VisitTypeListPage.java

```diff
@@ -21,7 +21,7 @@
 	private static final By ERROR = By.cssSelector("span.error");
 	private static final By VISIT_TYPE_LIST = By.cssSelector("#content form table tbody tr td");
 	private static final By RETIRED_VISIT_TYPE_LIST = By.cssSelector("#content form table tbody tr td del");
-	private static final By ALREADY_SAVED_VISIT_TYPE = By.cssSelector("#content > form > table > tbody > tr:nth-child(3) > td:nth-child(1) > a");
+	private static final By ALREADY_SAVED_VISIT_TYPE = By.xpath("//*[@id=\"content\"]/form/table/tbody/tr[3]/td[1]/a");
```

Comment:
What happens when you use classname, 🆔,  linktext

Reply:
@mherman22 it has no **classname**.. **id**

Reply:
and for **linktext** since after editing the text changes also it doesn't work

Reply:
@mherman22 it has no **classname** and, **id**, and for linktext since after editing the text changes its unable to retire and delete it.....therefore am gonna use cssSelector again but according to https://sqa.stackexchange.com/questions/27978/what-makes-a-good-selenium-locator

Reply:
xpaths should be the last resort when you cant find all others.

thanks for solving that in your recent commit